contracts/hyperlane-mailbox/src/main.sw

```diff
@@ -5,7 +5,17 @@ use std::{auth::msg_sender, bytes::Bytes, call_frames::contract_id, logging::log
 use std_lib_extended::bytes::*;
 
 use merkle::StorageMerkleTree;
-use ownership::{interface::Ownable, log_ownership_transferred, require_msg_sender};
+
+use ownership::{data_structures::State, only_owner, owner, set_ownership, transfer_ownership};
+
+abi Ownable {
```

Comment:
should we put this in `hyperlane-interfaces` so that we don't need to copy it everywhere?